istioctl/cmd/analyze.go

```diff
@@ -257,12 +265,16 @@ istioctl experimental analyze -k -d false
 			"Analyzers requiring resources made available by enabling service discovery will be skipped.")
 	analysisCmd.PersistentFlags().BoolVar(&colorize, "color", istioctlColorDefault(analysisCmd),
 		"Default true.  Disable with '=false' or set $TERM to dumb")
-	analysisCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Enable verbose output")
+	analysisCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false,
+		"Enable verbose output")
 	analysisCmd.PersistentFlags().Var(&failureLevel, "failure-threshold",
 		fmt.Sprintf("The severity level of analysis at which to set a non-zero exit code. Valid values: %v", diag.GetAllLevelStrings()))
 	analysisCmd.PersistentFlags().Var(&outputLevel, "output-threshold",
 		fmt.Sprintf("The severity level of analysis at which to display messages. Valid values: %v", diag.GetAllLevelStrings()))
-	analysisCmd.PersistentFlags().StringVarP(&msgOutputFormat, "output", "o", LogOutput, fmt.Sprintf("Output format: one of %v", msgOutputFormatKeys))
+	analysisCmd.PersistentFlags().StringVarP(&msgOutputFormat, "output", "o", LogOutput,
+		fmt.Sprintf("Output format: one of %v", msgOutputFormatKeys))
+	analysisCmd.PersistentFlags().StringVar(&meshCfgFile, "mesh-config-file", "",
+		"Overrides the mesh config values to use for analysis.")
```

Comment:
Can you describe what format the input is expected to be in the help comments (e.g. is this a kubernetes resource of kind ConfigMap or is it the actual contents of the ConfigMap)

Reply:
It's a YAML representing the mesh config object directly, not a k8s resource / configmap. This usage is consistent with how we're using it elsewhere, but admittedly the fact that it's a YAML that's not a K8s resouce is a bit confusing as well. Any ideas for specific wording that would be clearer?

Reply:
Ah, well if we use it elsewhere, then it seems fine. I guess as long as the error is reasonable when someone uses the ConfigMap resource instead...